src/UserSpaceInstrumentation/InjectLibraryInTracee.cpp

```diff
@@ -94,10 +99,20 @@ ErrorMessageOr<uint64_t> FindFunctionAddressWithFallback(
     return ErrorMessage(absl::StrFormat("Library does not exist at \"%s\"", path));
   }
 
-  // Figure out address of dlopen.
-  OUTCOME_TRY(auto&& dlopen_address,
-              FindFunctionAddressWithFallback(
-                  modules, {kDlopenInLibdl, kDlopenInLibc, kDlopenFallbackInLibc}));
+  // We will first look for dlmopen. If that function is not available we will fall back to dlopen.
+  // Both are handled in separate calls to FindFunctionAddressWithFallback because they have
+  // different function signatures.
```

Comment:
`dlmopen` was introduced in glibc 2.3.4 (Dec '04). So I'm not sure if we should even have the fallback to dlopen. WDYT?

It would simplify the code if we just try to call dlmopen and I don't see the advantage of not just doing that.

Reply:
Agree. Let's just use `dlmopen`

Reply:
Done.